crates/biome_css_analyze/tests/specs/nursery/noDuplicateSelectors/disallowInList.css

```diff
@@ -0,0 +1,19 @@
+/* valid cases: should not error */
```

Comment:
nit

You could add an [`:is()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:is) test case for future improvements to the rule.
i.e. `is:(a), a {}` could be detected one day